go/kbfs/libfuse/dir.go

```diff
@@ -723,13 +727,18 @@ func (d *Dir) Symlink(ctx context.Context, req *fuse.SymlinkRequest) (
 // Rename implements the fs.NodeRenamer interface for Dir.
 func (d *Dir) Rename(ctx context.Context, req *fuse.RenameRequest,
 	newDir fs.Node) (err error) {
+	oldNamePPS := d.node.ChildName(req.OldName)
+	// We need to log the new name before we have the new node, so
+	// just obfuscate it with the old node for now, it's the best we
+	// can do.
+	newNameLoggingPPS := d.node.ChildName(req.NewName)
 	ctx = d.folder.fs.config.MaybeStartTrace(ctx, "Dir.Rename",
 		fmt.Sprintf("%s %s -> %s", d.node.GetBasename(),
-			req.OldName, req.NewName))
+			oldNamePPS, newNameLoggingPPS))
 	defer func() { d.folder.fs.config.MaybeFinishTrace(ctx, err) }()
 
 	d.folder.fs.vlog.CLogf(
-		ctx, libkb.VLog1, "Dir Rename %s -> %s", req.OldName, req.NewName)
+		ctx, libkb.VLog1, "Dir Rename %s -> %s", oldNamePPS, newNameLoggingPPS)
```

Comment:
could we adopt a convention of putting a `*` at the end of filenames whose obfuscators might have the wrong conflict number mapping? e.g. here and in the CR code and maybe elsewhere?

Reply:
Ah, I just saw that you had the same `*` convention as I suggested in response to your other comment.  Great minds, etc etc.  As before, I'm not too worried about it.  Let's revisit if it becomes a problem and `deobfuscate` isn't as helpful as I think it will be.